specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -822,6 +974,10 @@
         }
       }
     },
+    "AdditionalCapability": {
+      "type": "string",
+      "description": "Enables or disables a capability on the Disk Pool"
```

Comment:
If only 2 status: enable/disable, this might be a bool.
Else please describe clearly what can be input with the string.

Reply:
We plan on using this field as a way for the customers to test out a new feature -- so its not a required field if they plan on creating a regular diskPool.

We decided to make it a `string` type instead of `object` to avoid changes to swagger while we're testing small feature flags internally

The lack of expected format for this field is intentional -- we were hoping to share the correct input with those who need it privately. Would be happy to answer any questions about this!

Reply:
In this way, could you clarify possible values in your description at least?